pyomo/contrib/incidence_analysis/incidence.py

```diff
@@ -16,8 +16,13 @@
 from pyomo.core.expr.visitor import identify_variables
 from pyomo.core.expr.numvalue import value as pyo_value
 from pyomo.repn import generate_standard_repn
+from pyomo.repn.plugins.nl_writer import AMPLRepnVisitor, AMPLRepn, text_nl_template
```

Comment:
It doesn't look like you're using the `text_nl_template` import in this file any more.

Reply:
Yes, other than `AMPLRepn`, these are now used by `config.py` instead. Good catch.